README.rst

```diff
@@ -17,7 +17,7 @@ composer.json:
 
     {
         "require": {
-            "guzzlehttp/oauth-subscriber": "0.4.*"
+            "guzzlehttp/oauth-subscriber": "0.5.*"
```

Comment:
`^0.5` is probably the syntax we should recommend (instead of `0.5.*`).

Reply:
👍